src/api/addresses.ts

```diff
@@ -0,0 +1,68 @@
+import {
+  BTC_LEGACY_DERIVATION,
+  BTC_SEGWIT_DERIVATION,
+  BTC_WRAPPED_SEGWIT_DERIVATION,
+  DEFAULT_ETH_DERIVATION,
+  HARDENED_OFFSET,
+} from '../constants';
+import { queue } from './utilities';
+
+export const fetchAddresses = async (
+  overrides?: GetAddressesRequestParams,
+): Promise<string[]> => {
+  return queue((client) =>
+    client
+      .getAddresses({
+        startPath: DEFAULT_ETH_DERIVATION,
```

Comment:
Not specific to this line, but please document default values like these. There should be a broader functional API docs page. Idk if the defaults should go there but they should be listed somewhere.